react-single-page-app/src/index.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+
+// <ms_docref_import_modules>
+import { Configuration,  PublicClientApplication } from "@azure/msal-browser";
+import { MsalProvider, useMsal, useIsAuthenticated } from "@azure/msal-react";
+// </ms_docref_import_modules>
+
+// <ms_docref_configure_msal>
+const configuration: Configuration = {
+  auth: {
+      clientId: "034ff080-d093-433d-915c-4f12ae03f100"
+  }
+};
+
+const clientInstance = new PublicClientApplication(configuration);
+// </ms_docref_configure_msal>
+
+// <ms_docref_get_graph_token>
+const getGraphToken =  async (msalInstance, accounts) => {
+  const tokenRequest = {
+    scopes: ["User.Read", "openid", "profile"],
```

Comment:
Should only need `User.Read` -- the rest are automatic (and not able to be changed anyhow)